crates/re_viewer/src/app.rs

```diff
@@ -562,6 +566,37 @@ impl App {
         }
     }
 
+    #[cfg(target_arch = "wasm32")]
+    fn run_copy_direct_link_command(&mut self, store_context: Option<&StoreContext<'_>>) {
+        let location = eframe::web::web_location();
+        let mut origin = location.origin;
+        if location.host == "app.rerun.io" {
+            // links to `app.rerun.io` can be made into permanent links:
+            let path = match &self.build_info.version.meta {
+                // not a final release, use commit hash
+                Some(..) => format!("commit/{}", &self.build_info.git_hash[..7]),
```

Comment:
it's a bit ugly to hard-code `..7` everywhere

```suggestion
                Some(..) => format!("commit/{}", &self.build_info.short_git_hash()),
```